app/src/js/utils/format.js

```diff
@@ -100,6 +100,18 @@ export const fromNowWithTooltip = (timestamp) => (
 const getIndicator = (prop) => {
   const indicator = {};
   switch (prop) {
+    case 'Completed':
+      indicator.color = 'success';
+      indicator.text = prop;
+      break;
+    case 'Failed':
+      indicator.color = 'failed';
+      indicator.text = prop;
+      break;
+    case 'Running':
+      indicator.color = 'orange';
```

Comment:
`orange` color is used for `missing` in GNF report as well.

Reply:
Right, I would probably dislike that also, but since you didn't do that originally, i wasn't going to ask you to go fix it. that's why suggested just adding another value to the _base.scss. Does that make sense?

Reply:
Yes, thanks for explain.  [fb93d01](https://github.com/nasa/cumulus-dashboard/pull/947/commits/fb93d0116a43460548238fdcdef78adc85fc5f58)